pkg/authorization/api/types.go

```diff
@@ -71,6 +77,29 @@ var (
 		KubeExposedGroupName:   {"pods", "replicationcontrollers", "serviceaccounts", "services", "endpoints", "persistentvolumeclaims", "pods/log"},
 		KubeAllGroupName:       {KubeInternalsGroupName, KubeExposedGroupName, QuotaGroupName},
 		KubeStatusGroupName:    {"pods/status", "resourcequotas/status", "namespaces/status"},
+
+		OpenshiftNonEscalatingViewableGroupName: {
```

Comment:
In the future you might want to have a test case that negates the set and verifies nothing changes (i.e verify with code that NonEscalating contains nothing from escalating